server/src/views/shell.js

```diff
@@ -9,9 +9,9 @@ exports.Shell = React.createClass({
         <title>
           PageShot
         </title>
-        <script src="/js/bundle.js" />
-        <link rel="stylesheet" href="/css/styles.css" />
-        <link rel="stylesheet" href="/css/login.css" />
+        <script src={"/js/bundle.js?git_revision=" + this.props.git_revision } />
+        <link rel="stylesheet" href={"/css/styles.css?git_revision=" + this.props.git_revision } />
+        <link rel="stylesheet" href={"/css/login.css?git_revision=" + this.props.git_revision }/>
```

Comment:
Maybe we can pass something like `props.linkify=function (link) {return link +(link.contains("?") ? "?" : "&") + "bust=" + gitRevision}` and then avoid the manual link construction here.